### ISSUE TITLE
Solving Сaptcha  by tampermonkey

### ISSUE THREAD

**Issue by SiisMax999** [BhaskarPanja93/View-Bot-Client#46]
I want to open via :tampermonkey
![Capt](https://user-images.githubusercontent.com/61063402/230829464-1088ffa3-0dc6-4a40-843c-96708033bb4c.PNG)

**Comment by BhaskarPanja93** [comment]
@SiisMax999 I have no clue about tampermonkey, and regarding the captcha, well this bot does it correctly, i dont think i need any other extensions. so thanks.

**Comment by SiisMax999** [comment]
How do I get over
I need
easier way
The old captcha earned a lot from it
I did it by putting code in tampermonkey
this code old captcha :+1: https://gist.github.com/origamiofficial/2557dd47fb0aaf08e3c298a236bfa14d

![Capture](https://user-images.githubusercontent.com/61063402/230913352-722adbd8-1994-46f5-bbaf-381ff196bfc1.PNG)
![Capture](https://user-images.githubusercontent.com/61063402/230913388-e573d1ba-91ef-4578-b438-05fe145c1201.PNG)

**Comment by BhaskarPanja93** [comment]
@SiisMax999 at this point the problem lies with Linkvertise, and not the captcha method, and we have nothing that we can do to fix it, only Linkvertise themselves can. so lets wait, and i'll do my part in updating the code as needed

**Comment by SiisMax999** [comment]
Try it try good luck to you

**Comment by Kirazt** [comment]
@SiisMax999 Could I pm you to ask something ? thanks

**Comment by DartMouthWrld** [comment]
> How do I get over I need easier way The old captcha earned a lot from it I did it by putting code in tampermonkey this code old captcha 👍 https://gist.github.com/origamiofficial/2557dd47fb0aaf08e3c298a236bfa14d
> 
> ![Capture](https://user-images.githubusercontent.com/61063402/230913352-722adbd8-1994-46f5-bbaf-381ff196bfc1.PNG) ![Capture](https://user-images.githubusercontent.com/61063402/230913388-e573d1ba-91ef-4578-b438-05fe145c1201.PNG)

@SiisMax999 How did you scale up? Are you using many physical computers?

**Comment by SiisMax999** [comment]
> > كيف يمكنني تجاوز الأمر أحتاج إلى طريقة أسهل لقد كسبت كلمة التحقق القديمة الكثير منها ، لقد فعلت ذلك عن طريق وضع رمز في tampermonkey هذا الكود القديم captcha👍 https://gist.github.com/origamiofficial/2557dd47fb0aaf08e3c298a236bfa14d
> > ![يأسر](https://user-images.githubusercontent.com/61063402/230913352-722adbd8-1994-46f5-bbaf-381ff196bfc1.PNG) ![يأسر](https://user-images.githubusercontent.com/61063402/230913388-e573d1ba-91ef-4578-b438-05fe145c1201.PNG)
> 
> @SiisMax999كيف صعدت؟ هل تستخدم العديد من أجهزة الكمبيوتر المادية؟

The method I use is different and easy and requires only one computer
use vpn
  USA only Very high when used, but now the captcha has been changed

**Comment by SiisMax999** [comment]
> @SiisMax999 Could I pm you to ask something ? thanks

yes pro ?

**Comment by softmachine1** [comment]
> > > كيف يمكنني تجاوز الأمر أحتاج إلى طريقة أسهل لقد كسبت كلمة التحقق القديمة الكثير منها ، لقد فعلت ذلك عن طريق وضع رمز في tampermonkey هذا الكود القديم captcha👍 https://gist.github.com/origamiofficial/2557dd47fb0aaf08e3c298a236bfa14d
> > > ![يأسر](https://user-images.githubusercontent.com/61063402/230913352-722adbd8-1994-46f5-bbaf-381ff196bfc1.PNG) ![يأسر](https://user-images.githubusercontent.com/61063402/230913388-e573d1ba-91ef-4578-b438-05fe145c1201.PNG)
> > 
> > 
> > @SiisMax999كيف صعدت؟ هل تستخدم العديد من أجهزة الكمبيوتر المادية؟
> 
> The method I use is different and easy and requires only one computer use vpn USA only Very high when used, but now the captcha has been changed

@SiisMax999  do you have twitter or discord ?👀

**Comment by SiisMax999** [comment]
> > > > كيف يمكنني تجاوز الأمر أحتاج إلى طريقة أسهل لقد كسبت كلمة التحقق القديمة الكثير منها ، لقد فعلت ذلك عن طريق وضع رمز في tampermonkey هذا الكود القديم captcha👍 https://gist.github.com/origamiofficial/2557dd47fb0aaf08e3c298a236bfa14d
> > > > ![يأسر](https://user-images.githubusercontent.com/61063402/230913352-722adbd8-1994-46f5-bbaf-381ff196bfc1.PNG) ![يأسر](https://user-images.githubusercontent.com/61063402/230913388-e573d1ba-91ef-4578-b438-05fe145c1201.PNG)
> > > 
> > > 
> > > @SiisMax999كيف صعدت؟ هل تستخدم العديد من أجهزة الكمبيوتر المادية؟
> > 
> > 
> > The method I use is different and easy and requires only one computer use vpn USA only Very high when used, but now the captcha has been changed
> 
> @SiisMax999 do you have twitter or discord ?👀

https://twitter.com/Beautif39251956

**Comment by DartMouthWrld** [comment]
@SiisMax999 Please explain how to scale. I can't dm your twitter. Or leave link to repo?

**Comment by SiisMax999** [comment]
> @SiisMax999يرجى شرح كيفية القياس. لا استطيع dm تويتر الخاص بك. أو ترك رابط الريبو؟

![Ct](https://user-images.githubusercontent.com/61063402/231084324-b550fc6c-faa8-406f-8db9-4793ee2054e9.PNG)

**Comment by Kirazt** [comment]
@SiisMax999 yep, i can't dm you on twitter either. Maybe you private your chat to stranger.

**Comment by SiisMax999** [comment]
> @SiisMax999 yep, i can't dm you on twitter either. Maybe you private your chat to stranger.

Send me accnt

**Comment by Kirazt** [comment]
@SiisMax999 https://twitter.com/KiraztB  here bro

**Comment by softmachine1** [comment]
@SiisMax999 you can dm me (https://twitter.com/soft_machine__)

**Comment by Luskafe** [comment]
@SiisMax999 drop your instagram bro i have some questions about scaling up earnings

**Comment by SiisMax999** [comment]
> @SiisMax999أسقط إنستغرام وإخوانه ، لدي بعض الأسئلة حول زيادة الأرباح

Send me Twitter. Pro

**Comment by softmachine1** [comment]
@SiisMax999 your twitter dms are closed

**Comment by KevinGeLe** [comment]
@SiisMax999 If you have problems with the linkvertise captcha, then I might start making a firefox extension. Since I already need an extension for Tor.

**Comment by SiisMax999** [comment]
> @SiisMax999إذا كانت لديك مشاكل مع linkvertise captcha ، فقد أبدأ في إنشاء امتداد لمتصفح فايرفوكس. لأنني بالفعل بحاجة إلى تمديد لتور.
Message me now on Twitter
https://twitter.com/Beautif39251956
Send me a script
  I want on tampermonkey

**Comment by softmachine1** [comment]
> > @SiisMax999إذا كانت لديك مشاكل مع linkvertise captcha ، فقد أبدأ في إنشاء امتداد لمتصفح فايرفوكس. لأنني بالفعل بحاجة إلى تمديد لتور.
> > Message me now on Twitter
> > https://twitter.com/Beautif39251956
> > Send me a script
> > I want on tampermonkey

@SiisMax999 you need to allow dms from others in the twitter settings

**Comment by SiisMax999** [comment]
> > > @SiisMax999إذا كانت لديك مشاكل مع linkvertise captcha ، فقد أبدأ في إنشاء امتداد لمتصفح فايرفوكس. لأنني بالفعل بحاجة إلى تمديد لتور.
> > > Message me now on Twitter
> > > https://twitter.com/Beautif39251956
> > > Send me a script
> > > I want on tampermonkey
> 
> @SiisMax999أنت بحاجة للسماح لـ dms من الآخرين في إعدادات twitter

try now

**Comment by Kirazt** [comment]
@SiisMax999 Yeah, it view the message box on your profile but after that you need to allow chat for stranger too. Follow me to test https://twitter.com/KiraztB

**Comment by KevinGeLe** [comment]
> > > > @SiisMax999إذا كانت لديك مشاكل مع linkvertise captcha ، فقد أبدأ في إنشاء امتداد لمتصفح فايرفوكس. لأنني بالفعل بحاجة إلى تمديد لتور.
> > > > Message me now on Twitter
> > > > https://twitter.com/Beautif39251956
> > > > Send me a script
> > > > I want on tampermonkey
> > 
> > 
> > @SiisMax999أنت بحاجة للسماح لـ dms من الآخرين في إعدادات twitter
> 
> try now

![image](https://user-images.githubusercontent.com/125448701/232090002-badeb754-39f3-4f63-8bcf-b44c60013947.png)

**Comment by SiisMax999** [comment]
> > > > > @SiisMax999إذا كانت لديك مشاكل مع linkvertise captcha ، فقد أبدأ في إنشاء امتداد لمتصفح فايرفوكس. لأنني بالفعل بحاجة إلى تمديد لتور.
> > > > > Message me now on Twitter
> > > > > https://twitter.com/Beautif39251956
> > > > > Send me a script
> > > > > I want on tampermonkey
> > > 
> > > 
> > > @SiisMax999أنت بحاجة للسماح لـ dms من الآخرين في إعدادات twitter
> > 
> > 
> > try now
> 
> ![image](https://user-images.githubusercontent.com/125448701/232090002-badeb754-39f3-4f63-8bcf-b44c60013947.png)
@Beautif39251956

**Comment by SiisMax999** [comment]
The arrival of the last withdrawal after changing the captcha
![Capture](https://user-images.githubusercontent.com/61063402/232178258-7e0a181d-e97d-424a-b59b-9cd58866d482.PNG)

**Comment by KevinGeLe** [comment]
> > > > > > @SiisMax999إذا كانت لديك مشاكل مع linkvertise captcha ، فقد أبدأ في إنشاء امتداد لمتصفح فايرفوكس. لأنني بالفعل بحاجة إلى تمديد لتور.
> > > > > > Message me now on Twitter
> > > > > > https://twitter.com/Beautif39251956
> > > > > > Send me a script
> > > > > > I want on tampermonkey
> > > > 
> > > > 
> > > > @SiisMax999أنت بحاجة للسماح لـ dms من الآخرين في إعدادات twitter
> > > 
> > > 
> > > try now
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/125448701/232090002-badeb754-39f3-4f63-8bcf-b44c60013947.png)
> > @Beautif39251956

I still cant send you a direct msg.

**Comment by softmachine1** [comment]
> > > > > > > @SiisMax999إذا كانت لديك مشاكل مع linkvertise captcha ، فقد أبدأ في إنشاء امتداد لمتصفح فايرفوكس. لأنني بالفعل بحاجة إلى تمديد لتور.
> > > > > > > Message me now on Twitter
> > > > > > > https://twitter.com/Beautif39251956
> > > > > > > Send me a script
> > > > > > > I want on tampermonkey
> > > > > 
> > > > > 
> > > > > @SiisMax999أنت بحاجة للسماح لـ dms من الآخرين في إعدادات twitter
> > > > 
> > > > 
> > > > try now
> > > 
> > > 
> > > ![image](https://user-images.githubusercontent.com/125448701/232090002-badeb754-39f3-4f63-8bcf-b44c60013947.png)
> > > @Beautif39251956
> 
> I still cant send you a direct msg.

@KevinGeLe dm me on @AzoFortnite75 I will explain you what he needs

**Comment by KevinGeLe** [comment]
> > > > > > > > @SiisMax999إذا كانت لديك مشاكل مع linkvertise captcha ، فقد أبدأ في إنشاء امتداد لمتصفح فايرفوكس. لأنني بالفعل بحاجة إلى تمديد لتور.
> > > > > > > > Message me now on Twitter
> > > > > > > > https://twitter.com/Beautif39251956
> > > > > > > > Send me a script
> > > > > > > > I want on tampermonkey
> > > > > > 
> > > > > > 
> > > > > > @SiisMax999أنت بحاجة للسماح لـ dms من الآخرين في إعدادات twitter
> > > > > 
> > > > > 
> > > > > try now
> > > > 
> > > > 
> > > > ![image](https://user-images.githubusercontent.com/125448701/232090002-badeb754-39f3-4f63-8bcf-b44c60013947.png)
> > > > @Beautif39251956
> > 
> > 
> > I still cant send you a direct msg.
> 
> @KevinGeLe dm me on @AzoFortnite75 I will explain you what he needs

Acceptos 
![image](https://user-images.githubusercontent.com/125448701/232258204-bb163587-f5a0-481f-9eb6-ba06127f8e69.png)
